### PR TITLE
Add keyboard keys to remove focus on input fields

### PR DESCRIPTION
**Pull request by dennis531**
Adds a keyboard event handler to remove the focus from input elements when the `esc` or `enter` key are pressed. This pull request is based on https://github.com/opencast/opencast-admin-interface/issues/643#issuecomment-2150046215.

Close https://github.com/opencast/opencast-admin-interface/issues/643

### CODE REVIEW

**Comment by dennis531** [comment]
> I don't understand the point of this pull request. There is already a key handler in RenderField.

This is true, but it doesn't apply to all field components, as not all of them have a component property. [In these cases, a simple HTML input will be used](https://formik.org/docs/api/field), which does not support Enter and ESC keys (at least in Firefox). For example, in the create user dialog, the RenderField or RenderMutlifield is not passed as component prop. 

To avoid redundant code, I created this wrapper that adds this key down handler to all Field components. If a component prop with value such as RenderField is passed, this handler will be ignored in the RenderField component.

File: src/components/shared/Field.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Field as FormikField } from "formik";
+import { FieldAttributes } from "formik/dist/Field";
+
+/**
+ * Wrapper for the Formik Fields
+ */
+export const Field = (props: FieldAttributes<any>) => {
```
Comment (lkiesow) [inline src/components/shared/Field.tsx:7]: How does this relate to
https://github.com/opencast/opencast-admin-interface/blob/172a3bf55585f8162a8ade14389507b18b7680c0/src/components/shared/wizard/RenderField.tsx#L38-L46

That seems to do something similar. Do they conflict? Should they be merged? Or is this just fine?

**Comment by lkiesow** [comment]
Then, do we still need the code in `RenderField` at all?

**Comment by dennis531** [comment]
> Then, do we still need the code in `RenderField` at all?

Yes, because the handler in `RenderField` manages the specific state of this component, which is different from the basic HTML input logik.

**Review by Arnei — APPROVED** [review]
While I can see developers accidentally importing the wrong Field component in the future, this does seem like the most reasonable solution to the given problem. Works and can be merged after the current merge conflicts are fixed imo.

**Comment by Arnei** [comment]
For stuff like the "Add User" oder "Create group" modals, it might be better if the `enter` key worked more like the `tab` key in that pressing it would automatically focus the next field. Not something that needs to be changed by this PR, certainly, I'm just musing here. I also have not thought about all UX or a11y implications yet.

**Pull request merged**